CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ the code was deployed.
 
 ### Added
 - Boron Sensor heartbeat optimizations, to improve debugging and door connectivity (CU-24qak6k).
+- Console function to enable developers to force reset and bypass DEVICE_RESET_THRESHOLD (CU-2e5adgd).
```

Comment:
Thanks for remembering.

Before this is merged, could you please squash your commits and rebase off `main`? Thanks!

Reply:
Im not sure how do to that. Usually i rebase and merge.....?

Reply:
The way I normally squash commits together is using the interactive rebase feature in git `git rebase -i HEAD~10` where 10 is the number of commits back in time that you now is more than you're going to need to modify. 

It will display all those 10 (or however many you specify) commits in the text editor with the word `pick` beside each one and a big set of instructions of what words you can change `pick` to in order to do other actions with it. 

Your most recent commit will be at the bottom. If you want to squash it with the commit directly above it in the list, you replace `pick` with `s` (or `squash`).

So in your case, because this whole PR should be a single commit, you can squash all the way back to your first commit (which will still say `pick`). 

Then you save the text file with all the `squash` and `pick` values and it will show you a combined version of the commit message to edit. Edit it down to what you'd like the commit message to be for this one new amazing commit and save it. 

Then if you look at your history using `git log`, you'll only see the one commit with the new message.

This will give us a nice, clean git history when it's committed into `main`. It will also make during the rebase from `main` easier because you will only have to deal with any conflicts once instead of a whole chain of them, once for each commit on your branch.

Like any rebase operation, this changes your git history, so when you next push to GitHub, you'll need to force push (`git push -f`). But don't do this until you're sure that you've got it the way you want it.

If you mess up at any point, try `rebase --abort` or just delete your branch locally and do a fresh `git pull` from GitHub of your branch and try again.

If you'd like to walk through it with me, we can do that too!

Reply:
Oh ok, cool. Yes if you could walk me through it that would be great! Thank you!